internal/config/config.go

```diff
@@ -81,7 +82,32 @@ type MTLS struct {
 	TLSSecretNamespace string `mapstructure:"secretNamespace"`
 }
 
-func LoadConfig(filePath string, fileName string) (*Config, error) {
+func LoadConfig() (*Config, error) {
+	v := viper.New()
```

Comment:
Can't we leverage the same viper "obj" or is it cleaner in this way?

Reply:
If I use the same Viper object (and thus I can reuse the same function), Viper will try to look for `configPath`(or was it `config-path`?) and `configName` in the YAML and fails because they are not optional yada yada...